scripts/latest-version/latest_version.go

```diff
@@ -103,12 +103,16 @@ func main() {
 	fmt.Print(util.TrimTagPrefix(tag))
 }
 
+func isReleaseBranch(branch string) bool {
+	return kgit.IsReleaseBranch(branch) && branch != kgit.Master
+}
+
```

Comment:
nit: can we have this below all calls to it? (underneath incVersion)

Reply:
Yes, I moved it below :+1: